packages/slate-base64-serializer/src/index.js

```diff
@@ -10,7 +54,7 @@ import { State } from 'slate'
 
 function encode(object) {
   const string = JSON.stringify(object)
-  const encoded = window.btoa(window.encodeURIComponent(string))
+  const encoded = btoa(window.encodeURIComponent(string))
```

Comment:
should remove window reference and use the globally defined `encodeURIComponent` directly, should work on node and browser

Reply:
yeah totally missed that window call